api/utils/keys/privatekey.go

```diff
@@ -214,6 +214,12 @@ func LoadPrivateKey(keyFile string) (*PrivateKey, error) {
 
 // ParsePrivateKey returns the PrivateKey for the given key PEM block.
 func ParsePrivateKey(keyPEM []byte) (*PrivateKey, error) {
+	//TODO(gzdunek): ParsePrivateKey has 60 usages, so I created ParsePrivateKeyWithPrompt
+	return ParsePrivateKeyWithPrompt(keyPEM, nil)
```

Comment:
Passing the `HardwareKeyPrompt` through so many layers is painful, especially in cases like this one, where `ParsePrivateKey` has so many usages. To make my life easier I created a new version of this function, to allow it specify a custom prompt. 
If the prompt is `nil`, I use the default `CLIPrompt`. I know it's a bit defensive technique, but idk if adjusting all the callsites would be better.